extensions/shared/external-media/sources/google-photos/filter-option.js

```diff
@@ -2,8 +2,14 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { SelectControl, Button } from '@wordpress/components';
+import {
+	SelectControl,
+	Button,
+	__experimentalNumberControl as BlockEditorNumberControl,
```

Comment:
simple question: are we safe using this experimental component across different core versions?